che-theia-plugins.yaml

```diff
@@ -309,7 +309,7 @@ plugins:
       mountSources: true
       volumeMounts:
         - name: mta-vscode 
-          path: "/home/theia/home/theia/redhat.rhamt-vscode-extension"
+          path: "/home/theia/mta/redhat.mta-vscode-extension"
```

Comment:
Is this change intentional? AFAIK the `/home/theia/home/theia` path is a workaround for plugins that hardcode such a path. Has that been fixed in your extension?

Reply:
@ericwill Yes, I have updated the path within the extension to use '/home/theia/mta/redhat.mta-vscode-extension'